materials/section-0/README.md

```diff
@@ -0,0 +1,228 @@
+
+
+# Introduction
+
+What is Clojure?
+
+>Clojure was forged in a mythic volcano by Rich Hickey. Using an alloy of Lisp, functional programming, and a lock of his own epic hair, he crafted a language that’s delightful yet powerful. Its Lisp heritage gives you the power to write code more expressively than is possible in most non-Lisp languages, and its distinct take on functional programming will sharpen your thinking as a programmer.
+
+\- CLOJURE for the BRAVE and TRUE
+
+That is a nice and short summary of Clojure. Although, some key things are missing from the description. As the description points out, Clojure is a Lisp, it's also dynamically typed, and it runs on the JVM. In the next section, we'll go through the fundamentals of the language.
+
+## Table of contents
+
+1. [Syntax](#syntax)
+2. [Data Types](#data-types)
+3. [Special Forms](#special-forms)
+4. [References](#references)
+
+## Syntax
+
+The syntax, or structure of the language is quite different from what you might be used too. Clojure is written as lists, or *forms*. The form is written as a pair of parenthesis, `()` which is also how you represent a list in Clojure. When you right a basic expression in Clojure, the first thing you will write is a pair of parenthesis, the first thing you write inside the parenthesis will *always be interpreted as a function to call*. The first position inside the parenthesis is called the function position. Anything after that, is interpreted as an argument(s) for the function you are calling. Any Clojure type that implements the `IFn` interface can be placed in the function position. This includes: `keyword`, `vector`, `map`, `set` and of course *functions* themselves, and last but not least, *macros* can also be placed in the function position.
```

Comment:
Some minor spelling corrections:
> from what you might be used **to**

> When you **write** a basic expression in Clojure

Reply:
Well, that was embarrassing. I swear there are more typos to come :/

Reply:
haha no sweat, 99% of my usefulness stems from speaking English natively